### PR TITLE
Remove more images not used by main application

### PR DESCRIPTION
**Pull request by mbj**

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb Ready for review, this was validated against the main app.

File: frontend/app/views/spree/orders/_line_item.html.erb [hunk 1]
```diff
@@ -30,7 +30,7 @@
       <%= line_item.display_amount.to_html unless line_item.quantity.nil? %>
     </td>
     <td class="cart-item-delete" data-hook="cart_item_delete">
-      <%= link_to image_tag('icons/delete.png'), '#', :class => 'delete', :id => "delete_#{dom_id(line_item)}" %>
+      <%= link_to 'delete', '#', :class => 'delete', :id => "delete_#{dom_id(line_item)}" %>
```
Comment (dkubb) [inline frontend/app/views/spree/orders/_line_item.html.erb]: Do you want to fix the hash syntax to use 1.9 Symbols?

**Comment by dkubb** [comment]
@mbj I did a pass on this. One tiny suggestion.

**Comment by mbj** [comment]
@dkubb Ready for the next pass.

**Comment by mbj** [comment]
@dkubb Ready for the next pass.

**Comment by dkubb** [comment]
@mbj I am fine with this change. Will merge when green.

**Pull request merged**